src/code/z_draw.c

```diff
@@ -519,14 +519,14 @@ void GetItem_DrawSkullToken(PlayState* play, s16 drawId) {
 }
 
 void GetItem_DrawEggOrMedallion(PlayState* play, s16 drawId) {
-    s32 pad;
+    DrawItemTableEntry* entry = &sDrawItemTable[drawId];
 
     OPEN_DISPS(play->state.gfxCtx, "../z_draw.c", 772);
 
     Gfx_SetupDL_26Opa(play->state.gfxCtx);
     gSPMatrix(POLY_OPA_DISP++, MATRIX_NEW(play->state.gfxCtx, "../z_draw.c", 776), G_MTX_MODELVIEW | G_MTX_LOAD);
-    gSPDisplayList(POLY_OPA_DISP++, sDrawItemTable[drawId].dlists[0]);
-    gSPDisplayList(POLY_OPA_DISP++, sDrawItemTable[drawId].dlists[1]);
+    gSPDisplayList(POLY_OPA_DISP++, entry->dlists[0]);
+    gSPDisplayList(POLY_OPA_DISP++, entry->dlists[1]);
```

Comment:
ftr this pattern with the temp also seems to match (but isn't required) in the other `GetItem_Draw...` functions